code-of-conduct.md

```diff
@@ -1,113 +1,78 @@
+These emails are sent to the US-RSE Code of Conduct Enforcement Committee.
+You may also contact committee members individually in order to initiate a report. 
 
-If you believe someone is violating the code of conduct, please report this in a timely manner (see How to Submit a Report). Code of
-conduct violations reduce the value of the community for everyone. The US-RSE Association members take reports of misconduct very seriously
-and are committed to preserving and maintaining the welcoming nature of our community.
+You will be asked the following questions as part of the reporting process:
 
-All complaints will be reviewed and investigated and will result in a response that is deemed necessary and appropriate to the
-circumstances.
-All reports will be kept confidential, with the exception of cases where the Steering Committee determines the report should be shared with law enforcement.
-In those cases, the report will be shared with the proper legal authorities.
+- Who performed the violating action?
+  If not known, please supply details to help identify them where possible.
+- What was the violating action?
+- How should we contact you in response to this report?
 
-In some cases the US-RSE Leadership may determine that a public statement will need to be made. If that’s the case, the identities of
-all involved parties and reporters will remain confidential unless those individuals instruct us otherwise.
+The reporting party/parties will be kept confidential.
 
-For possibly unintentional breaches of the code of conduct, you may want to respond to the person and point out this code of conduct 
-(either in public or in private, whatever is most appropriate). If you would prefer not to do that, please report the issue (see How to 
-Submit a Report) or contact a US-RSE Association Steering Committee member for assistance.
+### What to Expect After You Report
 
-Take care of each other. Alert the US-RSE Association Steering Committee (contact@us-rse.org) if you notice a dangerous situation, 
-someone in distress, or violations of this code of conduct, even if they seem inconsequential.
+The US-RSE Association is committed to promptly addressing any reported issues.
+You will be contacted as quickly as possible by a member of the committee acknowledging your report, and shortly thereafter be appointed a case lead who will be your point of contact throughout the report response process.
+The Case Lead may ask additional questions for clarification.
+After information has been gathered, the Case Lead will inform you of the committee’s decision and the consequences of any violations of the Code of Conduct.
 
-### How to Submit a Report
-If you feel your safety is in jeopardy or the situation is an emergency, contact local law enforcement before making a report to the US-RSE Association. (In the U.S., dial 911.)
+There will be a private record made of the case, to be accessible to all current and future committee members.
```

Comment:
Might add clarification: "besides recused member(s)"?

Reply:
This makes sense, though I'm unsure how it could be implemented